Sources/RealDeviceMap/API/ApiRequestHandler.swift

```diff
@@ -964,7 +964,7 @@ class ApiRequestHandler {
                 "type": pokestopOptionsString
             ])
 
-            for i in 1...4 {
+            for i in 1...5 {
                 let pokestopLure = Localizer.global.get(value: "filter_pokestop_lure_\(i)")
```

Comment:
missing 
```                </label>
                <label class="btn btn-sm btn-size select-button-new" data-id="\(i)"
                 data-type="pokestop-lure" data-info="huge">
                <input type="radio" name="options" id="show" autocomplete="off">\(hugeString)
                </label>```

Reply:
i think thats it, i looked over everything and it seems to be what i have im just not sure about index as its different then mine, i tested with a dummy lure in database too and it seemed to work, just need to actual real data to now

Reply:
this is only one row each. the change from `for i in 1...4` to `for i in 1...5` should be enough.
what you have duplicated is, that you added one more column for sizing information for the icon on map (small, normal, large, huge)

Reply:
thank you wasnt too sure how that worked, ill fix on mine :)